attest/core/Cargo.toml

```diff
@@ -19,6 +19,7 @@ std = [
     "failure/std",
     "mbedtls/std",
     "sha2/std",
+    "chrono",
```

Comment:
Even though `chrono` supports `no_std` and this would've been ideal here, the fact that we need it for the `build-dependencies` with `std` prevents us from not gating it with a feature flag due to cargo's feature unification :(

Reply:
This cargo nightly feature should fix that: https://doc.rust-lang.org/cargo/reference/unstable.html#features

```
 -Zfeatures=host_dep
```

Reply:
With latest rust, we can also enable that feature from Cargo.toml, AFAIK.

Reply:
However, we're currently not on latest Rust. We have two options:
1) Waiting with this until we switch to latest Rust. This is a blocker for MC-1458, unless we want this code duplicated in every service that has an enclave.
2) Proceed with it as presented in this PR, and fix with https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#resolver when we upgrade Rust.
How would you like to proceed? I'm happy either way and can look into upgrading Rust before focusing on this (which might be higher priority anyway).

I believe `host_dep` is not available in our current version:
```
% cargo -Z help

Available unstable (nightly-only) flags:

    -Z avoid-dev-deps   -- Avoid installing dev-dependencies if possible
    -Z minimal-versions -- Install minimal dependency versions instead of maximum
    -Z no-index-update  -- Do not update the registry, avoids a network request for benchmarking
    -Z unstable-options -- Allow the usage of unstable options such as --registry
    -Z config-profile   -- Read profiles from .cargo/config files
    -Z timings          -- Display concurrency information
    -Z doctest-xcompile -- Compile and run doctests for non-host target using runner config

Run with 'cargo -Z [FLAG] [SUBCOMMAND]'
```